NEWS

```diff
@@ -1,22 +1,45 @@
-x/y/2022 ola-0.10.9
+x/y/2023 ola-0.10.9
  Features:
- *
+ * Python 3 compatibility across the board (including the RDM Responder Tests)!
+   #1506
+ * Support for the JMS USB2DMX PRO V2.1 device via the FTDI plugin #1728
 
  API:
  * Python: Add a fetch current DMX example.
 
  RDM Tests:
- * 
+ * Python 3 compatibility of the RDM Tests #1599
+ * Fix a longstanding bug in the GetMaxPacketSize RDM test around timeouts
```

Comment:
any reference? PR# or commit?

Reply:
It was #1736 but looking at NEWS we've historically not bothered putting PR numbers in, just issue numbers when people reported it. I think I probably just caught this by inspection or when doing something else.

Reply:
I've taken them out for now and I'll put them in the other PR so they just appear at the correct time...